libpretixprint/src/main/java/eu/pretix/libpretixprint/templating/Layout.java

```diff
@@ -23,6 +24,7 @@
 
 import static com.lowagie.text.Utilities.millimetersToInches;
 import static com.lowagie.text.Utilities.millimetersToPoints;
+import static eu.pretix.libpretixprint.templating.TextUtilsKt.linesRequiredHardBreak;
```

Comment:
I cannot say much about the actual code, but reading it I think this method should be named `linesRequireHardBreak` (without the d) or `linesHardBreakRequired`.